### DISCUSSION TITLE
Mod startup failure; am I missing something?

### DISCUSSION THREAD

**Discussion by GenkiDenkiz** [tsunamods-codes/7th-Heaven#163] [Q&A] [answered]
So I handled this like I would modding most other games, I launched FF7 raw, played a little, then got into this modding stuff. I'm using the latest (3.2.0) build, installed via the installer. I have the game via Steam, and I install my games outside of system files (custom Steam directory). I pointed 7th Heaven at my "ff7_en.exe" and it told me it couldn't use it and made a new executable under some 1.02 patch or something, I thought whatever, it probably knows what its doing. I gave it a library folder within the ff7 install folder, installed my mods, made sure my load order was good. Went to launch with mods.. Get this:

"### Launching Final Fantasy VII ###
Checking System DEP status...
Checking FF7 is not running ...
Checking FF7 .exe exists at E:\Gmaes\Steam\steamapps\common\FINAL FANTASY VII\ff7.exe ...
Verifying installed game is compatible ...
Verifying game is not installed in a System/Protected folder ...
Creating missing required directories ...
Verifying English game files exist ...
Verifying game is full/max install ...
Verifying additional files for 'battle' and 'kernel' folders exist ...
Verifying all movie files exist ...
Verifying music files exist ...
Verifying latest game driver is installed ...
FFNx seems to be present.
Verifying game driver shaders folders exist ...
Verifying ff7 exe ...
Checking a profile is active ...
Checking mod compatibility requirements ...
Checking mod constraints for compatibility ...
Checking mod load order requirements ...
Looking for game disc ...
Unknown error when launching the game: Could not load file or assembly 'System.IO.FileSystem.DriveInfo, Version=7.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified."

I've got no idea what that means honestly, any clues? Thanks in advance, I'm new to this.

EDIT:
Figured I should mention I'm running Windows 11 22H2 and may as well just list specs ahead of time
Intel(R) Core(TM) i7-9750H 2.60GHz
NVIDIA GeForce RTX 2060 (Laptop)

**Comment by JuakinBM** [comment]
I'm getting same error, can't point to FF7.EXE (since Steam version does not have one) when given FF7_Launcher.exe (if I give ff7_es.exe does nothing and always starts the game without mods) then displays that message

"it couldn't use it and made a new executable under some 1.02 patch or something"

But then I get a message that the Disc is missing... even if it is mounted correctly...

**Comment by julianxhokaxhiu** [comment, marked as answer]
It is faster to join Discord and troubleshoot it over there then here. Looks like something on your setup side is wrong. Cheers
